docs/finding-schema.md

```
@@ -13,6 +13,37 @@ Canonical normalized finding fields:
 - `fingerprint`: deterministic SHA-256 key (hex)
 - `firstSeenAt` / `lastSeenAt`: RFC3339 timestamps
 - `metadata`: optional source extras
+- `routing` (optional): owner/status/SLA/escalation/done-state extension for Active Findings UX
+
+### Optional `routing` extension
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)